### PR TITLE
[power manager] concrete facilities and levels for auto scheme are added

### PR DESCRIPTION
**Pull request by milchakov**
кого еще добавить в pr - пока не знаю

### CODE REVIEW

**Comment by milchakov** [comment]
@rokuz PTAL

File: map/power_managment/power_manager.hpp [hunk 1]
```diff
@@ -58,18 +41,16 @@ class PowerManager
   {
     DECLARE_VISITOR(visitor(m_facilities, "current_state"), visitor(m_scheme, "scheme"));
 
-    FacilitiesState m_facilities = {{true, true}};
+    Config() { m_facilities.fill(true); }
```
Comment (rokuz) [inline map/power_managment/power_manager.hpp]: : m_facilities(true) {}

**Comment by milchakov** [comment]
JTA

**Comment by milchakov** [comment]
JTIOS

**Pull request merged**